src/lib/serviceRoutines/postContextEntitiesByEntityId.cpp

```diff
@@ -41,6 +40,10 @@
 * postContextEntitiesByEntityId - 
 *
 * POST /ngsi9/contextEntities/{entityId}
```

Comment:
Having two ones is better. All de documention headers should be udpated. Not sure if in this PR (creating a new issue if we defer this for the future?)